src/engraving/libmscore/timesig.cpp

```diff
@@ -601,6 +602,24 @@ QString TimeSig::accessibleInfo() const
     return QString("%1: %2").arg(EngravingItem::accessibleInfo(), timeSigString);
 }
 
+void TimeSig::added()
+{
+    IF_ASSERT_FAILED(score()) {
+        return;
+    }
+
+    score()->fixTicks();
```

Comment:
Does time signature affect this? I thought not.

Reply:
Unfortunately, the current implementation of former Score::fixTicks includes lots of time signature calculations and building the timesigMap. I think we'll be able to extract this method and use it separately soon